doc/nixpkgs-committers.md

```diff
@@ -3,7 +3,17 @@
 There are over 200 Nixpkgs committers with access to merge any PR.
 To become a Nixpkgs committer, you can apply [in this issue](https://github.com/NixOS/nixpkgs/issues/321665).
 
-The [Nixpkgs committer delegation team](./github.md#nixpkgs) is given the responsibility and authority of changing the list of [Nixpkgs committers](https://github.com/orgs/NixOS/teams/nixpkgs-committers).
+The [Nixpkgs committer delegation team](#team) is given the responsibility and authority of changing the list of [Nixpkgs committers](https://github.com/orgs/NixOS/teams/nixpkgs-committers).
+
+## Team
+
+Nixpkgs committer delegation team consists of these people:
+
+-- [@Mic92](https://github.com/Mic92)
+-- [@NickCao](https://github.com/NickCao)
+-- [@jtojnar](https://github.com/jtojnar) ([Matrix](https://matrix.to/#/@jtojnar:matrix.org), [Discourse](https://discourse.nixos.org/u/jtojnar))
+
+They can be contacted either individually, or through a [group message on Discourse](https://discourse.nixos.org/g/nixpkgs-admins).
```

Comment:
Team name is a misnomer

Reply:
This was chosen due to Discourse handle length limitation (20 characters). Other suggestions welcome.

Reply:
Hmm, how about:
- nixpkgs-nominations
- nixpkgs-bit-givers
- nixpkgs-perm-mgmt
- nixpkgs-write-mgmt


Undortunately too long:
- nixpkgs-committer-mgmt
- nixpkgs-perm-grantors